### DISCUSSION TITLE
Multiple Desktop Environments

### DISCUSSION THREAD

**Discussion by Komodoro** [rocksdanister/lively#933] [Ideas]
Hi, quick idea: 

Intro:
Windows, with the recent OS release (W11), has introduced (AFAIK) the possibility to have different backgrounds for w/e Desktop Environment you are in.

Idea:
Have the same possibility. But have the background on pause (due to memory issues) until you switch it or just support a limit number of Desktop Environment simultaneously. Since you can with multiple screens, I dunno if it goes the same way but it could be close. 

Best regards
Komodo

p.S. awsome app. congrats.

**Comment by rocksdanister** [comment]
This right? https://github.com/rocksdanister/lively/issues/808

**Reply by Komodoro** [reply]
right [488](https://github.com/rocksdanister/lively/issues/488).

I prefered to open a discussion about it then an issue. Since it wasn't a proper feature request but more of a insight debate on the possibility for the matter. And I only noticed after that those threads existed. So my bad for not deleting them afterwards.
